edx_proctoring/constants.py

```diff
@@ -46,7 +46,7 @@
 SOFTWARE_SECURE_CLIENT_TIMEOUT = (
     settings.PROCTORING_SETTINGS['SOFTWARE_SECURE_CLIENT_TIMEOUT'] if
     'SOFTWARE_SECURE_CLIENT_TIMEOUT' in settings.PROCTORING_SETTINGS
-    else getattr(settings, 'SOFTWARE_SECURE_CLIENT_TIMEOUT', 30)
+    else getattr(settings, 'SOFTWARE_SECURE_CLIENT_TIMEOUT', 60)
```

Comment:
@dianakhuang: One thought is you add a commit now for the final constants.  In other words, revert this back to 30 and make `DEFAULT_CLIENT_POLLING_INTERVAL` and `REDUCED_CLIENT_POLLING_INTERVAL` both 5.  It may be quick to get this landed and do the more complex testing with the other constant changes in the other story, rather than doing complex testing twice.

Thoughts?